docs/setup-iptables.md

```diff
@@ -3,6 +3,11 @@
 In order to prevent the indy-node process from reaching of open file descriptors limit caused by clients connections it is strongly
```

Comment:
What about `CLIENT_CONNECTIONS_LIMIT=10000` in `postinst_node`?

Reply:
Sure.